### ISSUE TITLE
Loading a GeoTIFF DEM (32 bits) file is not working

### ISSUE THREAD

**Issue by joa-quim** [f3d-app/f3d#1303]
Windows and 2.3.0 version here.

When I try to load a GeoTIFF DEM file (means each note contains a float32 value) I get a _vertical curtain_ like the one shown in the screen-capture. I was expecting to see a 3D view of a relief surface. Am I wrong in my expectations or something is not working right?

![image](https://github.com/f3d-app/f3d/assets/537321/527337b9-00db-48ea-b180-0668fb51d434)

**Comment by mwestphal** [comment]
Hi @joa-quim 

Can you share your file ?

Please use the issue template when possible.

**Comment by joa-quim** [comment]
OK, that file was a bit big so I cropped a smaller region that is attached bellos. To be clear, what I was expecting to see is something like the next fig that was made with another 3D viewer (note, I added a the shaded relief effect to better exemplify the expected outcome).

(Thanks for the instantaneous reply)
 
![image](https://github.com/f3d-app/f3d/assets/537321/1bb0a261-e80f-4bc3-9132-bc7129ccddf4)


[sagres.tif.zip](https://github.com/f3d-app/f3d/files/14526747/sagres.tif.zip)

**Comment by mwestphal** [comment]
Indeed, it looks like VTK is not reading this file the way you expect it. I have no idea why yet, more investigation needed.

I have the same results in F3D and in ParaView as well. More investigation needed.

**Comment by joa-quim** [comment]
Thanks again. In case it's useful, I read those type of files with [GDAL](https://gdal.org/index.html) that can read a lot of different raster (and vector too) formats. Being able to load these type of data in F3D would be very nice.

**Comment by mwestphal** [comment]
Indeed, GDAL seems to be able to open these files.
